plugins/usertiming.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * @module UserTiming
+ * @desc
+ * Plugin to collect metrics from the W3C User Timing API.
+ * For more information about User Timing,
+ * see: http://www.w3.org/TR/user-timing/
+ *
+ * This plugin is dependent on the UserTimingCompression library
+ * see: https://github.com/nicjansma/usertiming-compression.js
+ * UserTimingCompression must be loaded before this plugin's init is called.
+ */
+
+/*global UserTimingCompression*/
+
+(function() {
+
+	BOOMR = BOOMR || {};
+	BOOMR.plugins = BOOMR.plugins || {};
+	if (BOOMR.plugins.UserTiming) {
+		return;
+	}
+
+	var impl = {
+		complete: false,
+		initialized: false,
+		supported: false,
+		options: {"from": 0, "window": BOOMR.window},
+
+		/**
+		 * Calls the UserTimingCompression library to get the compressed user timing data
+		 * that occurred since the last call
+		 *
+		 * @returns {string} compressed user timing data
+		 */
+		getUserTiming: function() {
+			var timings, res, now = BOOMR.now();
+			var utc = UserTimingCompression || BOOMR.window.UserTimingCompression;
```

Comment:
where is this global defined? Should we refer to it as `window.UserTimingCompression` instead? I've seen some older versions of Opera throw an exception when we try this.